db/data/test-data/users.js

```diff
@@ -1,24 +1,24 @@
 module.exports = [
   {
-    username: 'butter_bridge',
-    name: 'jonny',
+    username: "butter_bridge",
+    name: "jonny",
     avatar_url:
-      'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg'
+      "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg",
   },
   {
-    username: 'icellusedkars',
-    name: 'sam',
-    avatar_url: 'https://avatars2.githubusercontent.com/u/24604688?s=460&v=4'
+    username: "icellusedkars",
+    name: "sam",
+    avatar_url: "https://avatars2.githubusercontent.com/u/24604688?s=460&v=4",
   },
   {
-    username: 'rogersop',
-    name: 'paul',
-    avatar_url: 'https://avatars2.githubusercontent.com/u/24394918?s=400&v=4'
+    username: "rogersop",
+    name: "paul",
+    avatar_url: "https://avatars2.githubusercontent.com/u/24394918?s=400&v=4",
   },
   {
-    username: 'lurker',
-    name: 'do_nothing',
+    username: "lurker",
+    name: "do_nothing",
     avatar_url:
-      'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png'
-  }
+      "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+  },
 ];
```

Comment:
This was an accident.

Reply:
Linters eh? 🤷

Reply:
Make sure you're being selective with your git staging